### DISCUSSION TITLE
Flatpak version has delete button greyed out.

### DISCUSSION THREAD

**Discussion by caj411** [qarmin/czkawka#1135] [General]
I'm using the Flatpak version and the delete button stays grey even when I select files.  I cannot delete any files.
Does anyone else have this problem or is it specific to me?  Deleting dups is the second most important function behind identifying them.

**Comment by qarmin** [comment]
I installed latest 6.1 version as first flatpak package and everything works fine

https://github.com/qarmin/czkawka/assets/41945903/5447a93c-0cee-402b-a3cf-519d2920fdac

**Reply by caj411** [reply]
Thanks. I appreciate getting back to me.
It must be something odd on my end. I'll try and figure it out. Strange.
